test/test-ncu.js

```diff
@@ -51,7 +51,7 @@ describe('npm-check-updates', function () {
         it('should read --packageFile', function() {
             var tempFile = 'test/temp_package.json';
             fs.writeFileSync(tempFile, '{ "dependencies": { "express": "1" } }', 'utf-8')
-            return spawn('node', ['bin/npm-check-updates', '--jsonUpgraded', '--packageFile', tempFile])
+            return spawn('node', ['bin/ncu', '--jsonUpgraded', '--packageFile', tempFile])
```

Comment:
had to change; else, `JSON.parse` fails because of the extra `[INFO]` output that gets printed (since role of `bins` have [swapped](https://github.com/tjunnone/npm-check-updates/pull/203/files#r51649427)